inc/template-tags.php

```diff
@@ -11,8 +11,10 @@
 	 * Prints HTML with meta information for the current post-date/time.
 	 */
 	function twentynineteen_posted_on() {
+		/* translators: 1: post date in Full Date/Time. 2: post date in F j, Y. */
 		$time_string = '<time class="entry-date published updated" datetime="%1$s">%2$s</time>';
```

Comment:
There is no internationalization function used here the translator comment is not needed.